src/main/java/org/sagebionetworks/web/client/jsinterop/EntityHeaderTableProps.java

```diff
@@ -0,0 +1,79 @@
+package org.sagebionetworks.web.client.jsinterop;
+
+import java.util.ArrayList;
+import java.util.List;
+import jsinterop.annotations.JsFunction;
+import jsinterop.annotations.JsNullable;
+import jsinterop.annotations.JsOverlay;
+import jsinterop.annotations.JsPackage;
+import jsinterop.annotations.JsType;
+import org.sagebionetworks.repo.model.Reference;
+import org.sagebionetworks.repo.model.request.ReferenceList;
+import org.sagebionetworks.web.client.utils.CallbackP;
+
+@JsType(isNative = true, namespace = JsPackage.GLOBAL, name = "Object")
+public class EntityHeaderTableProps extends ReactComponentProps {
+
+  ReferenceJsObject[] references;
+  boolean isEditable;
+
+  @FunctionalInterface
+  @JsFunction
+  public interface OnUpdateReferenceList {
+    void onUpdate(ReferenceJsObject[] referenceListJsObject);
+  }
+
+  @FunctionalInterface
+  @JsFunction
+  public interface OnUpdateString {
+    void onUpdatString(String newValue);
```

Comment:
This typo shouldn't matter since this is a functional interface
```suggestion
    void onUpdateString(String newValue);
```